lckernel/cad/geometry/geospline.h

```diff
@@ -90,7 +93,7 @@ namespace lc {
                  * @return bool closed
                  */
                 bool closed() const;
-                const std::vector<std::vector<lc::geo::Coordinate>> beziers() const;
```

Comment:
Is it possible to keep this variable and initialise it in the constructor by calling Spline::beziers()? This way they don't get re-created each time we ask for a vector of Beziers. It will take a bit more memory, but I think this will speed up onder processing within the app for example during drawing and/or intersection calculations. Let me know your reasoning why you want to create the vector for each call rather then storing it. ~This is open for different thoughts (memory vs performance).